openrr-apps/src/robot_client.rs

```diff
@@ -89,13 +90,28 @@ impl RobotClient {
             &all_joint_trajectory_clients,
             &full_chain_for_collision_checker,
         );
+
+        let mut joints_poses: HashMap<String, HashMap<String, Vec<f64>>> = HashMap::new();
+        for joints_pose in &config.joints_poses {
+            if !joints_poses.contains_key(&joints_pose.client_name) {
+                joints_poses.insert(joints_pose.client_name.to_owned(), HashMap::new());
```

Comment:
Is it difficult to use entry to make it simple (You might not think it is simpler)?

```rust
joints_poses.entry(joints_pose.client_name.clone()).or_insert(HashMap::new()).insert(xx, yyy)
```